sqlmesh/core/test/result.py

```diff
@@ -1,10 +1,17 @@
 from __future__ import annotations
 
 import types
+import typing as t
 import unittest
 
 
 class ModelTextTestResult(unittest.TextTestResult):
+    successes: t.List[unittest.TestCase]
```

Comment:
do we need this here?

Reply:
I added it for mypy. I think successes needs to be annotated somewhere, but it could also be done in the __init__ function. I figured here made it more apparent there's a `successes` attribute for this class.